README.md

```diff
@@ -1,2 +1,45 @@
-# checkpoint-03
-[ checkpoint, ruby ]
+# Checkpoint 03
+
+To get started, fork and clone this repository locally.
+
+> Commit after each question
+
+## Ruby
+
+1. Assign the variable `x` to the value 5.
+
+```ruby
+# Your answer here
+
+```
+
+2. Update the code, so it can run successfully to print `"Blink 182"` to the console
+   ​
+```ruby
+# Your answer here
+band = "Blink" + 182
+
+```
+
+3. Given the string `"ABC"`  print to the console the output `"cbacbacba"`
```

Comment:
imagine people will `puts cbacbacba` should they dosomething else? `.downcase.reverse * 3`